src/App.js

```diff
@@ -1,26 +1,94 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import AddFood from './components/AddFood'
+import Foodbox from './components/FoodBox'
+
+class App extends Component {
+
+  state = {
+    foodList: foods,
+    filteredList: foods,
+    showForm: false
+  }
+
+
```

Comment:
Try to avoid this empty segments. A single empty line is enough to separate sections.